homeassistant/components/overkiz/number.py

```diff
@@ -34,6 +36,37 @@ class OverkizNumberDescription(NumberEntityDescription, OverkizNumberDescription
     """Class to describe an Overkiz number."""
 
     inverted: bool = False
+    set_native_value: Callable[
+        [float, Callable[..., Awaitable[None]]], Awaitable[None]
+    ] | None = None
+
+
+async def _async_set_native_value_boost_mode_duration(
+    value: float, execute_command: Callable[..., Awaitable[None]]
+) -> None:
+    """Update the boost duration value."""
+
+    if value > 0:
+        await execute_command(OverkizCommand.SET_BOOST_MODE_DURATION, value)
+        await asyncio.sleep(1)  # wait one second to not overload the device
+        await execute_command(
+            OverkizCommand.SET_CURRENT_OPERATING_MODE,
+            {
+                OverkizCommandParam.RELAUNCH: OverkizCommandParam.ON,
+                OverkizCommandParam.ABSENCE: OverkizCommandParam.OFF,
+            },
+        )
+    else:
+        await execute_command(
+            OverkizCommand.SET_CURRENT_OPERATING_MODE,
+            {
+                OverkizCommandParam.RELAUNCH: OverkizCommandParam.OFF,
+                OverkizCommandParam.ABSENCE: OverkizCommandParam.OFF,
+            },
+        )
+
+    await asyncio.sleep(3)  # wait 3 seconds to have the new duration in
```

Comment:
It would be nice if these values were constants so you can patch them out when this eventually gets tests

Reply:
Fixed in d9a44f3965f4d49a5d91d43c84e53999bae4db4c